lib/models/tag.js

```diff
@@ -8,17 +8,25 @@ const MixinSpecificationExtensions = require('../mixins/specification-extensions
 
 /**
  * Implements functions to deal with a Tag object.
+ * 
  * @class
  * @alias module:@asyncapi/parser#Tag
- * @extends Base
+ * @augments Base
  * @mixes MixinDescription
  * @mixes MixinExternalDocs
  * @mixes MixinSpecificationExtensions
  * @returns {Tag}
  */
 class Tag extends Base {
   /**
-   * @returns {string}
+   * @param {...any} args arguments
+   * @constructor
+   */
+  constructor(...args) {
+    super(...args);
+  }
```

Comment:
@derberg I did the necessary change here to make sure the sonar cloud bug doesn't come up, but I think it's not the possible workaround for this issue. 
Also, I did check that the same bug which is being reported [here](https://sonarcloud.io/project/issues?id=asyncapi_parser-js&pullRequest=305&resolved=false&types=BUG) is also mentioned in the [master branch](https://sonarcloud.io/project/issues?id=asyncapi_parser-js&resolved=false&types=BUG) but we have ignored it. 
perhaps this is a false positive and we need to ignore this check for this repo.